lib/dgram.js

```diff
@@ -121,6 +122,12 @@ function Socket(type, listener) {
 
   // If true - UV_UDP_REUSEADDR flag will be set
   this._reuseAddr = options && options.reuseAddr;
+  
+  if (options && options.recvBufferSize)
```

Comment:
Technically, we could use the libuv calls to with `value === 0` to get the current value.

Also, it looks like these underscored properties aren't used in `setRecvBufferSize()` and `setSendBufferSize()`.